### PR TITLE
Addition of check for CosmosDB Local Authentication

### PR DESCRIPTION
**Pull request by c0mmad0r3**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

Custom Policy ID: CKV_AZURE_140
Custom Policy Name: Ensure that Local Authentication is disabled on CosmosDB
Custom Policy IaC Type: Terraform
Custom Policy type and Provider: AzureRM
IaC configuration documentation: https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/cosmosdb_account#local_authentication_disabled

Additional Information: https://docs.microsoft.com/en-us/azure/cosmos-db/how-to-setup-rbac#disable-local-auth

### CODE REVIEW

File: checkov/terraform/checks/resource/azure/CosmosDBLocalAuthDisabled.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        # These are the terraform objects supported by this check (ex: aws_iam_policy_document)
+        supported_resources = ['azurerm_cosmosdb_account']
+
+        # Valid CheckCategories are defined in checkov/common/models/enums.py
+        categories = [CheckCategories.IAM]
+        super().__init__(name=description, id=id, categories=categories, supported_resources=supported_resources)
+
+    def get_inspected_key(self) -> str:
+        return "local_authentication_disabled"
+    def get_expected_value(self):
```
Comment (gruebel) [inline checkov/terraform/checks/resource/azure/CosmosDBLocalAuthDisabled.py:22]: just add one line break here

**Review by gruebel — COMMENTED** [review]
thanks looks good, just the one thing the linter found

**Comment by c0mmad0r3** [comment]
@gruebel added line breaks between 21 & 22 as requested.  Please take a look.

**Review by gruebel — APPROVED** [review]
thanks great job 🥇

**Pull request merged**